common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -54,7 +54,8 @@
             this.sr_container = this.$('.sr-is-focusable');
             this.num_contents = this.contents.length;
             this.id = this.el.data('id');
-            this.ajaxUrl = this.el.data('ajax-url');
+            this.getCompletionUrl = runtime.handlerUrl(element, 'get_completion');
```

Comment:
I think this is a part I don't have a great understanding of. The runtime already has a method defined to handle this? I don't see any other changes related.

Reply:
I don't fully understand this concept of a runtime either. It's a versioned JS specific runtime?